styles/styles.css

```
@@ -308,8 +341,16 @@ p {
     line-height: 21px;
     text-align: center;
     color: #333333;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin-top: 25px;
+    margin-left: 274,5px;
+    margin-bottom: 69px;
+    margin-right: 274,5px;
```

```suggestion
    color: #333333;
```

Всем параграфам не нужен флекс и вот эти отступы. 
Это слишком рано.